internal/app/api/handler/user/user.go

```diff
@@ -119,7 +140,7 @@ func (h *Handler) GetListUsers(w http.ResponseWriter, r *http.Request) {
 
 	userList, err := h.srv.GetListUsers(r.Context(), pageParameter, sizeParameter)
 	if err != nil {
-		respond.JSON(w, http.StatusUnauthorized, h.em.InvalidValue.Request)
+		respond.JSON(w, http.StatusInternalServerError, h.em.InvalidValue.Request)
```

Comment:
use if err:= ...; err {} incase no new variable is created
Log error before response